flag_engine/features/models.py

```diff
@@ -26,9 +26,10 @@ class MultivariateFeatureOptionModel:
 
 @dataclass
 class MultivariateFeatureStateValueModel:
-    id: int
     multivariate_feature_option: MultivariateFeatureOptionModel
     percentage_allocation: float
+    id: int = None
```

Comment:
I decided not add a django id because it's in prod already and  load model to schema will  fail if it the model does not have an id what do you think?

Reply:
Yep, that's fine for now I think.